src/test/java/com/connexta/store/StoreITests.java

```diff
@@ -223,10 +219,6 @@ void testRetrieveDataNotFound() {
   @Test
   void testIngestRequest() throws Exception {
```

Comment:
❓ I don't see a test for `addMetadata` in this class, but I'd expect there to be one here where we stub the indexServer.

Reply:
I removed it since the flow is ingest should be called. Since we expect a file to always be present before calling addMetadata, there are issues with calling this method before ingest.

Reply:
So there are no itests for addMetadata? quarantine has this same problem. I think we need a test for addMetadata to make sure that everything is wired up correctly.

Reply:
No. AddMetadata is exercised through Ingest. Ingest tests cannot pass without addMetadata working.

Reply:
Ingest is the primary work flow and entry point into CDR. In fact I should probably probably modify the quarantine test to have transform return a response to trigger it rather than call quarantine directly.

Reply:
Oh, I see that you stub the transform status server in ingest so that the poller will call addMetadata 👍.